lib/pg_saurus/create_index_concurrently.rb

```diff
@@ -70,7 +70,6 @@ module Migration
     #
     # @see ActiveRecord::ConnectionAdapters::SchemaStatements.add_index in pg_saurus gem
     def add_index(table_name, column_name, options = {}, &block)
-      table_name = ::ActiveRecord::Migrator.proper_table_name(table_name)
```

Comment:
proper_table_name moved into ActiveRecord::Migration. Now you can use
```
table_name = proper_table_name(table_name)
```
http://stackoverflow.com/questions/26622488/activerecordmigrator-proper-table-name-is-deprecated